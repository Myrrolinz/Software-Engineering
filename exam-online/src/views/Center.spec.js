import { shallowMount } from '@vue/test-utils';
import Center from './Center';





describe('<Center/>', () => {
	const wrapper = shallowMount(Center);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(Center);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

  /*
  it("slide correctly", () => {
		// 滑动 slide
		wrapper.find('slide-verification').trigger('check-result');
	  });
  		// 可以立即获取 msg 最新的值，不再需要 wrapper.vm.$nextTick();
		expect(wrapper.find('h1').text())
		  .toEqual('new message');
		  */
