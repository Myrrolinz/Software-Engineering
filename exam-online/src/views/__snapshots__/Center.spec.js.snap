// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Center/> snapshot测试 1`] = `
"<div class=\\"center\\">
  <el-form-stub status-icon=\\"\\" model=\\"[object Object]\\" rules=\\"[object Object]\\" label-width=\\"80px\\">
    <h1>个人中心</h1>
    <el-form-item-stub label=\\"姓名\\" prop=\\"name\\">
      <el-input-stub></el-input-stub>
    </el-form-item-stub>
    <el-form-item-stub label=\\"性别\\" prop=\\"gender\\">
      <el-select-stub placeholder=\\"请选择性别\\">
        <el-option-stub value=\\"m\\" label=\\"男\\"></el-option-stub>
        <el-option-stub value=\\"f\\" label=\\"女\\"></el-option-stub>
      </el-select-stub>
    </el-form-item-stub>
    <el-form-item-stub label=\\"班级\\" prop=\\"clazz\\">
      <el-select-stub placeholder=\\"请选择年级\\"></el-select-stub>
    </el-form-item-stub>
    <el-form-item-stub>
      <el-button-stub type=\\"primary\\" icon=\\"\\" nativetype=\\"button\\">确认修改</el-button-stub>
      <el-button-stub type=\\"default\\" icon=\\"\\" nativetype=\\"button\\">取消</el-button-stub>
    </el-form-item-stub>
  </el-form-stub>
</div>"
`;
